ibis/backends/dask/udf.py

```diff
@@ -223,7 +221,7 @@ def execute_analytic_node_groupby(op, *args, aggcontext, **kwargs):
         func = op.func
         groupings = args[0].index
         parent_df = args[0].obj
-        out_type = op._output_type.to_dask()
+        out_type = op.return_type.to_dask()
```

Comment:
for naming consistency - I think it's better to call this `output_type`  since it's more common in ibis codebase

Reply:
`output_type` is a method of the operation classes, I think that was the main reason to use an underscored alternative.